database/seeds/August16ArticleSeeder.php

```diff
@@ -1,188 +0,0 @@
-        ],
-        [
-            'magazine' => 'August 2016',
-            'author' => 'Reza Lavaryan',
-            'website' => 'SitePoint',
-            'category' => 'Library',
-            'url' => 'https://www.sitepoint.com/framework-agnostic-php-cronjobs-made-easy-with-crunz/',
-            'title' => 'Framework-Agnostic PHP Cronjobs Made Easy with Crunz!',
-            'publish_date' => '2016-08-24',
-            'description' => 'In the latest post on sitepoint, Reza tells us managing cron jobs using Crunz library.',
-        ],
-    ];
-
-    /**
-     * Run the database seeds.
-     *
-     * @return void
-     */
-    public function run()
-    {
-        $this->addWebsites();
-        $this->addAuthors();
-        $this->addArticles();
-    }
-
-    /**
-     * Add Articles
-     */
-    protected function addArticles()
-    {
-        foreach ($this->articles as $article) {
-            $articleModel = new Article();
-
-            $magazine = Magazine::where('name', $article['magazine'])->first();
-            $articleModel->magazine_id = $magazine->id;
-
-            //ToDO: There could be two authors with same name. Handle them.
-            $author = Author::where('name', $article['author'])->first();
-            $articleModel->author_id = $author->id;
-
-            $website = Website::where('name', $article['website'])->first();
-            $articleModel->website_id = $website->id;
-
-            $category = Category::where('name', $article['category'])->first();
-            $articleModel->category_id = $category->id;
-
-            $articleModel->url = $article['url'];
-            $articleModel->title = $article['title'];
-            $articleModel->published_at = $article['publish_date'];
-            $articleModel->description = $article['description'];
-            $articleModel->created_at = Carbon::now()->format('Y-m-d H:i:s');
-            $articleModel->save();
-
-            $magazine->no_of_articles++;
-            $magazine->save();
-        }
-    }
-
-    /**
-     * Add addWebsites
-     */
-    protected function addWebsites()
-    {
-        foreach ($this->websites as $website) {
-            $websiteModel = new Website();
-            $websiteModel->name = $website['name'];
-            $websiteModel->site_url = $website['siteUrl'];
-            $websiteModel->feed_url = $website['feedUrl'];
-            $websiteModel->save();
-        }
-    }
-
-    /**
-     * Seed authors
-     */
-    protected function addAuthors()
-    {
-        foreach ($this->authors as $author=>$website) {
-            $authorModel = new Author();
-            $authorModel->name = $author;
-
-            $websiteModel = Website::where('name', $website)->first();
-            $authorModel->save();
-            $authorModel->websites()->attach($websiteModel);
-            $authorModel->save();
-        }
-    }
-}
```

Comment:
That is separate PR. Why it is showing as part of this PR?

These changes are not related to login form.

This probably happened due to wrong branch creation/merges.